ui/appui-react/src/appui-react/shared/ActionButtonItemDef.tsx

```diff
@@ -90,3 +89,5 @@ export abstract class ActionButtonItemDef extends ItemDefBase {
     );
   }
 }
+
+import { ActionItemButton } from "../toolbar/ActionItemButton";
```

Comment:
should be able to revert this change

Reply:
Reverting this leads to a circular dependency. Is there a problem w/ this?

Reply:
gotcha, i wasnt sure if this was by accident or circular dep. long term we probably want to fix this

Reply:
i _think_ this change here is breaking the viewer and probably anyone using appui-react with the 3.5 rc
investigating...
<img width="953" alt="image" src="https://user-images.githubusercontent.com/11051042/205365983-7580e74a-5e53-4c67-b7e7-70a8bb02ff1c.png">

Reply:
able to repro on master w/ ui-test-app.
unable to repro w/ `acb7d2654d9533eb3e898c628951f43b15ef07ff` (commit right before this pr went through)